src/User.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class User
+{
+    protected string $username;
+    protected string $hashed_pass;
+    protected string $bio;
+    protected string $avatar;
+
+    public function getUsername(): string
+    {
+        return $this->username;
+    }
+
+    public function getAvatar(): string
+    {
+        return $this->avatar;
+    }
+
+    public function getBio(): string
+    {
+        return $this->bio;
+    }
+
+    public function getHashedPass(): string
```

Comment:
public function getHashedPass(): string keeps slipping into the code 😭 must remove pls

Reply:
🤯 hopefully it is gone gone now...